azure-pipelines.yml

```diff
@@ -95,8 +95,7 @@ jobs:
       - bash: |
           source activate qcodes &&
           cd qcodes &&
-          pytest -m "not serial" --junitxml=test-parallel-results.xml --cov=qcodes --cov-report=xml --cov-config=.coveragerc --hypothesis-profile ci &&
-          pytest -n 0 --dist no -m serial --junitxml=test-serial-results.xml --cov-append --cov=qcodes --cov-report=xml --cov-config=.coveragerc --hypothesis-profile ci
+          pytest --junitxml=test-parallel-results.xml --cov=qcodes --cov-report=xml --cov-config=.coveragerc --hypothesis-profile ci &&
```

Comment:
it should be the other way around , right? the windows part should still run serial/not-serial

Reply:
Yes don't know how that happened :(